### ISSUE TITLE
Add screenshots on F-Droid

### ISSUE THREAD

**Issue by Korb** [dftec-es/planetcon#7]
There are currently no screenshots on [the app's page in the F-Droid catalog](https://f-droid.org/en/packages/com.dftec.planetcon/index.html). Without a preview, only by a text description, it is difficult to decide whether to download and install the application or not. I propose to add a few screenshots that most fully describe the appearance and functionality of the application.

**Comment by dftec-es** [comment]
I agree, but I did not create the f-droid entry and I can't modify it to add screenshots.
Thanks for the suggestion anyway.
